apps/package.json

```diff
@@ -75,18 +73,13 @@
     "phantomjs": "wjordan/phantomjs#v2.0.0",
     "react": "0.13.3",
     "react-tools": "0.13.3",
```

Comment:
Do we still need react-tools?  Will was saying something about it being used by reactify, but deprecated now.